src/main/java/frc/robot/commands/gpm/ShootKnownPos.java

```diff
@@ -0,0 +1,61 @@
+package frc.robot.commands.gpm;
+
+import edu.wpi.first.wpilibj2.command.ParallelCommandGroup;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.commands.ArmToPos;
+import frc.robot.constants.ArmConstants;
+import frc.robot.subsystems.gpm.Arm;
+import frc.robot.subsystems.gpm.Shooter;
+import frc.robot.subsystems.gpm.StorageIndex; 
+
+public class ShootKnownPos extends SequentialCommandGroup {
+	
+	/**
+	* A preset shot that can be taken by the {@link frc.robot.commands.gpm.ShootKnownPos} command
+	*/
+	public enum ShotPosition {
+		// TODO: add actual values
+		SUBWOOFER_TOP(0.0, 0.0),
+		SUBWOOFER_MIDDLE(0.0, 0.0),
+		SUBWOOFER_BOTTOM(0.0, 0.0);
```

Comment:
For the shooter, these should all be the same. You can probably leave them for now until we figure out what that position is in the next PR.

Reply:
small chance they might be different as well. Leave for now, fix later if neccessary